apps/sms/locales/sms.en-US.properties

```diff
@@ -42,7 +42,7 @@ remove-subject         = Remove subject
 to                         = To:
 new                        = New
 message                    = Message
-messagesInput.placeholder  = Message
+messagesInput.title        = Message
 messagesSubjectInput_placeholder = Subject
```

Comment:
I realize we still have this one too; but this uses a really different approach (adding/removing a real element when there is/isn't a value -- see subject_composer.js).